src/styles/_utils/reset.scss

```diff
@@ -0,0 +1,15 @@
+html,
+body,
+main,
+div,
+h1,
+h2,
+h3,
+h4,
+ul,
+li,
+p,
+a {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
If you want to reset styles for a large number of tags, maybe it's better to do it via `*`?